composer.json

```diff
@@ -29,7 +29,8 @@
     },
     "require-dev": {
 
-        "phing/phing": "2.14.0"
+        "phing/phing": "2.14.0",
```

Comment:
So for clarification, do I need to manually remove unused files or can phing do that? Or do we need to remove phing from vendor and composer.json if we are not using it?